Atomic/rpm_host_install.py

```diff
@@ -132,7 +150,8 @@ def rm_add_files_to_host(old_installed_files_checksum, exports, prefix="/", file
                         shutil.copystat(src_file, dest_path)
```

Comment:
Question: will the comment in `_copyfile` apply here as well? /me does not know very well  about copystat tho. Seems like the selinux handle is very similar to `_copyfile` part right?
```
# we cannot use shutil.copy2() or shutil.copystat() here as it would override the
# security.selinux xattr.
```

Reply:
great catch.  I pushed a fixup patch to change it to `copymode`